### ISSUE TITLE
AS3 Policy_Action Forward Traffic with SNAT options 

### ISSUE THREAD

**Issue by mlbenedict7** [F5Networks/f5-appsvcs-extension#489]
Trying to Enable the Source Address Translation SNAT option but the Schema Reference doesn't indicate a way to set that flag.

Here's an example of what I'd like to do but doesn't seem to trigger the SNAT Option flag in the GUI.  
```json
            "snat_test_policy": {
                "class": "Endpoint_Policy",
                "strategy": "first-match",
                "rules": [
                    {
                        "name": "snat_test_tcp_80",
                        "conditions": [
                            {
                                "type": "httpUri",
                                "path": {
                                    "operand": "starts-with",
                                    "values": [
                                        "/anything-api"
                                    ]
                                }
                            }
                        ],
                        "actions": [
                            {
                                "type": "forward",
                                "event": "request",
                                "snat": {
                                    "enable": "true"
                                },
                                "select": {
                                    "pool": {
                                        "use": "dummy_app_http_80_pool"
                                    }
                                }
                            }
                        ]
                    }
                ]
            }
```
Attached is the screenshot of the Options in the GUI.

Thanks

![Screenshot 2021-08-11 161946](https://user-images.githubusercontent.com/78821470/129097624-b7e140d5-9269-45b3-be49-d20a129885bf.jpg)

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added this request to our internal product backlog as AUTOTOOL-2727.

**Comment by dstokesf5** [comment]
This has been resolved in the 3.33.0 release.
